source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst

```diff
@@ -20,36 +19,32 @@ Wiggle battery harness connector. Often these are loose because the screws loose
 Apply considerable force onto the battery cable at 90 degrees to try to move the direction of the cable leaving the battery, if successful the connection was not tight enough to begin with and it should be redone.
 
 Secure the battery to robot connection
-------------------------------------------------------------------
+--------------------------------------
 
 .. image:: images/preemptive-troubleshooting/preCheckConnecc.png
-   :width: 400
 
 In almost every event we see at least one robot where a not properly secured battery connector (the large Anderson) comes apart and disconnects power from the robot. This has happened in championship matches on the Einstein and everywhere else. Its an easy to ensure that this doesn't happen to you by securing the two connectors by wrapping a tie wrap around the connection. 10 or 12 tie wraps for the piece of mind during an event is not a high price to pay to guarantee that you will not have the problem of this robot from an actual event after a bumpy ride over a defense.
 
 120 Amp circuit breaker
-------------------------------------
+-----------------------
 
 .. image:: images/preemptive-troubleshooting/preCheckBreaker.png
-   :width: 400
 
 Apply a twisting force onto the cable to rotate the harness.  If you are successful then the screw is not tight enough.  Split washers might help here, but in the mean time, these require checking every few matches.
 
 Because the metal is just molded into the case, every once in awhile you will break off the bolt, ask any veteran team and they’ll tell you they go through a number of these every few seasons.  After tightening the nut, retest by once again trying to twist the cable.
 
 Power Distribution Panel (PDP)
-------------------------------------
+------------------------------
 
 .. image:: images/preemptive-troubleshooting/preCheckPDP.png
-   :width: 700
 
 Make sure that split washers were placed under the PDP screws, but it is not easy to visually confirm, and sometimes you can’t.  You can check by removing the case.    Also if you squeeze the red and black wires together, sometimes you can catch the really lose connections.
 
 Tug test everything
-------------------------------------
+-------------------
 
 .. image:: images/preemptive-troubleshooting/preCheckTug.png
```

Comment:
This should be replaced as well.

Reply:
Does the image I added in 5100856 work well?